docs/platform/howto/use-aws-privatelinks.rst

```diff
@@ -1,4 +1,4 @@
-Using AWS PrivateLink with Aiven services
+Use AWS PrivateLink with Aiven services
 =========================================
```

Comment:
to avoid warnings or errors, I'd suggest to have the underline exactly where the sentence ends. So in this case, you need to remove two characters of this `=` similar as you removed in the title.